packages/discord.js/src/structures/GuildAuditLogsEntry.js

```diff
@@ -14,6 +14,7 @@ const Partials = require('../util/Partials');
 const { flatten } = require('../util/Util');
 
 const Targets = {
+  /** @deprecated this was never used or a possible value to begin with */
```

Comment:
If this is to target main, this can just be removed and a separate pull request can be made to backport it into v14.

Otherwise, we can keep this as-is, change the base branch to v14, and open another pull request for main.